website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -250,3 +250,239 @@ The GET endpoints above can optionally return related resources, if requested wi
 | ------------------ | -------------------------------------- |
 | `run`              | The run this policy check belongs to.  |
 | `run.workspace`    | The associated workspace of the run.   |
+
+<!-- BEGIN: TFC:only name:opa-policies -->
+
+## List Policy Evaluations in the Task Stage
```

Comment:
```suggestion
## List OPA Policy Evaluations in the Task Stage
```

Reply:
Also @mrinalirao This just occurred to me. Are any of the endpoints on this page now Sentinel Only? If so, we should mark them as such, so users know which ones to use for the policy set type they have :)